Please fix error

Simulator ServiceWorker registration successful with scope:  https://trg-maker.userpxt.io/
pxtapp.js:1 webusb: get devices
pxtapp.js:1 error: pxt_modules/ws2812b/main.ts(3,50): error TS2304: Cannot find name 'DigitalPin'.
error: pxt_modules/ws2812b/main.ts(7,40): error TS2304: Cannot find name 'DigitalPin'.

--- a/main.ts
+++ b/main.ts
@@ -1,10 +1,16 @@
 namespace ws2812b {
     //% shim=sendBufferAsm
-    export function sendBuffer(buf: Buffer, pin: DigitalPin) {
+    /**
+     * Sends a color buffer to a light strip
+     */
+    export function sendBuffer(buf: Buffer, pin: number) {
     }
 
     //% shim=setBufferMode
-    export function setBufferMode(pin: DigitalPin, mode: number) {
+    /**
+     * Sets the buffer mode
+     */
+    export function setBufferMode(pin: number, mode: number) {
 
     }
 

--- a/pxt.json
+++ b/pxt.json
@@ -1,6 +1,6 @@
 {
     "name": "ws2812b",
-    "version": "0.1.2",
+    "version": "0.1.3",
     "description": "A driver for WS2812B programmable LEDs in MakeCode",
     "license": "MIT",
     "dependencies": {

